### PR TITLE
Remove FromJSON functions (Part 1)

### PR DESCRIPTION
**Pull request by agnivade**
```release-note
Removed the following functions:
AccessDataFromJson
AccessResponseFromJson
AnalyticsRowFromJson
AnalyticsRowsFromJson
AuditFromJson
AuditsFromJson
AuthDataFromJson
AuthorizeRequestFromJson
BotFromJson
BotPatchFromJson
BotListFromJson
```

### CODE REVIEW

**Comment by streamer45** [comment]
Tests are not very happy about this :D

**Comment by agnivade** [comment]
Yeah, some basic mistakes from my side. Will fix.

File: api4/bot.go [hunk 1]
```diff
@@ -30,8 +31,9 @@ func (api *API) InitBot() {
 }
 
 func createBot(c *Context, w http.ResponseWriter, r *http.Request) {
-	botPatch := model.BotPatchFromJson(r.Body)
-	if botPatch == nil {
+	var botPatch *model.BotPatch
+	err := json.NewDecoder(r.Body).Decode(&botPatch)
+	if err != nil {
```
Comment (streamer45) [inline api4/bot.go:36]: You really don't like the short form do you? :stuck_out_tongue:

**Review by streamer45 — APPROVED** [review]
So good to see far more deletions than insertions :tada: Less is more they say.

File: api4/bot.go (hunk 1 above)
Comment (agnivade) [reply api4/bot.go:36]: It's not intentional :laughing: That's just how I think of writing the code.

**Pull request merged**